core/templates/dev/head/editor/feedback_tab/FeedbackTab.js

```diff
@@ -89,6 +89,54 @@ oppia.controller('FeedbackTab', [
     });
   };
 
+  // TODO(Allan): Implement ability to edit suggestions before applying.
+  $scope.showSuggestionModal = function() {
+    $modal.open({
+      templateUrl: 'modals/editorViewSuggestion',
+      backdrop: true,
+      size: 'lg',
+      resolve: {
+        suggestion: function() {
+          return $scope.activeThread.suggestion;
+        }
+      },
+      controller: [
+        '$scope', '$modalInstance', 'suggestion',
+        function($scope, $modalInstance, suggestion) {
+        $scope.oldContent = suggestion.state_content.old_content;
+        $scope.newContent = suggestion.state_content.new_content;
+
+        $scope.acceptSuggestion = function() {
+          $modalInstance.close({
+            action: 'accept'
+          });
+        };
+
+        $scope.rejectSuggestion = function() {
+          $modalInstance.close({
+            action: 'reject'
+          });
+        };
+
+        $scope.cancelReview = function() {
+          $modalInstance.dismiss('cancel');
+        };
+      }]
+    }).result.then(function(result) {
+      threadDataService.resolveSuggestion($scope.activeThread.suggestion.suggestion.id, result.action);
```

Comment:
Line too long. Break line at '(' and indent subsequent line by 2; feel free to break at comma subsequently if needed.